### PR TITLE
2.3.x fix stale event cache

### PR DESCRIPTION
**Pull request by jamestalton**
Signed-off-by: James Talton <jtalton@redhat.com>

https://github.com/stolostron/backlog/issues/19561

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -22,6 +22,11 @@
         "find-my-way": "^4.3.0",
         "node-fetch": "^2.6.1",
         "pino": "^6.11.3",
+        "event-stream": "^4.0.1",
+        "get-value": "^3.0.1",
+        "got": "^11.8.3",
+        "pluralize": "^8.0.0",
+        "proxy-agent": "^5.0.0",
```
Comment (KevinFCormier) [inline backend/package.json:29]: Is `proxy-agent` used in 2.3?

Comment (jamestalton) [reply backend/package.json]: fixed

**Pull request merged**